frontend/src/ui/Card.tsx

```diff
@@ -1,9 +1,9 @@
-import { FiAlertTriangle, FiInfo } from "react-icons/fi";
+import { FiAlertTriangle, FiInfo, FiSunrise } from "react-icons/fi";
 import { match } from "../util";
 
 
 type Props = JSX.IntrinsicElements["div"] & {
-    kind: "error" | "info";
+    kind: "error" | "info" | "welcome";
```

Comment:
This seems a bit specific for this rather generic component. :thinking: :shrug:

Reply:
Absolutely, had the same thought. But couldn't come up with a solution that's not vastly more work. Any idea?
But also, I don't super mind.

Reply:
Dunno if this qualifies as vastly more work already, but one idea I had was to extract all of the decisions that are made based on this enum into a data structure which you could pass in. That would contain sth like the icon used, background color, text color, ... (or for simplicity's sake just a whole CSS thing) :shrug: